src/Relations/BelongsToThrough.php

```diff
@@ -385,7 +348,7 @@ public function getFirstForeignKeyName()
      */
     public function getQualifiedFirstLocalKeyName()
     {
-        /** @var TIntermediateModel $lastThroughParent */
+        /** @var \Illuminate\Database\Eloquent\Model $lastThroughParent */
```

Comment:
I'm not sure these inline `@vars` are necessary? They should be avoided wherever possible